src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/RoutingResolverPass.php

```diff
@@ -11,27 +11,17 @@
 
 namespace Symfony\Bundle\FrameworkBundle\DependencyInjection\Compiler;
 
-use Symfony\Component\DependencyInjection\Reference;
-use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+@trigger_error(sprintf('The %s class is deprecated since version 3.3 and will be removed in 4.0. Use Symfony\Component\Routing\DependencyInjection\RoutingResolverPass instead.', RoutingResolverPass::class), E_USER_DEPRECATED);
```

Comment:
you can move that line below the "use" and use `BaseRoutingResolverPass::class` in `sprintf`